CommandLine/protos/server.proto

```diff
@@ -65,3 +66,15 @@ message SyntaxCheckRequest {
   optional int32 script_count = 2;
   repeated string script_names = 3;
 }
+
+message System {
```

Comment:
It's unclear to me what this represents; maybe add a doc comment. Is this, eg, `Widget_Systems`, and then `subsystems` includes `GTK+`? I'd call this, eg, `SystemType`. Probably less easily confused with a builtin that way. 😛

Reply:
Fair enough: eb57ccfbb53a0cb53d7552a5b1a187e86490c908

I also added some documentation comments to the messages.